src/app/recommended-communities/recommended-communities.component.spec.ts

```diff
@@ -57,6 +61,32 @@ const communities: CommunityResponse[] = [
   } as CommunityResponse
 ];
 
+const authenticatedUser: UserIdentity = {
+  userId: 'e6b808eb-b6bd-447d-8dce-3e0d66b17759',
+  userName: 'tester',
+  firstName: 'Toni',
+  lastName: 'Tester',
+  email: 'toni.tester@myskills.io',
+  roles: ['ROLE_USER']
+};
+
+const currentUser = {
+  id: authenticatedUser.userId,
+  userName: authenticatedUser.userName,
+  firstName: authenticatedUser.firstName,
+  lastName: authenticatedUser.lastName,
+  email: authenticatedUser.email,
+  coach: false,
+};
+
+const communityUserRegistrations: CommunityUserRegistrationResponse[] = [
+  {
+    user: currentUser,
+    approvedByUser: true,
+    approvedByCommunity: true
```

Comment:
Isn't it better to set `approvedByCommunity` to `null` here? The authenticated user sends a request to join closed community and that means the request is not approved by the community yet.

Reply:
thank you. done